.github/actions/test-ruby/action.yml

```diff
@@ -44,11 +47,19 @@ runs:
       env:
         ENV_PATH: ${{ inputs.ENV_PATH }}
 
+    - name: Split the input variable into an array
+      if: inputs.RUBY_RAKE_COMMANDS != null
+      env:
+        arr: ${{ inputs.RUBY_RAKE_COMMANDS }}
+      run: |
+        IFS=',' read -ra arr <<< "$arr"
+        for i in "${arr[@]}"; do
+          $i
```

Comment:
We may want to wrap this with a check to see if the command starts with `ruby` or `rails` to limit execution of arbitrary commands. Something like this might work:

```

if [[ $i == rails* ]] || [[ $i == rake* ]]; then
  $i
fi
```

Reply:
good catch. will limit it to bundle